osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNoteTail.cs

```diff
@@ -18,6 +19,11 @@ public partial class DrawableHoldNoteTail : DrawableNote
 
         protected internal DrawableHoldNote HoldNote => (DrawableHoldNote)ParentHitObject;
 
+        /// <summary>
+        /// The minimum uncapped result for a late release.
+        /// </summary>
+        public HitResult LateReleaseResult { get; set; } = HitResult.Miss;
```

Comment:
I personally have huge objections to this existing and will resist very hard to this getting anywhere near merge with this member present. It would be in the best interests of this pull to try and eliminate the need for it.

Reply:
How to apply changes to hit results? Is there any existing mechanism or should I make a new mod hook?

Reply:
Closest thing we currently have would be the osu! mod Strict Tracking that subclasses DHOs to get the desirable result.